src/index.html

```diff
@@ -1,12 +1,344 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page__content">
+    <header class="menu">
+      <div class="menu__container">
+        <a href="#home" class="menu__logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#apple">apple</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#samsung">samsung</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#smartphones">smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#computers"
+                data-qa="nav-hover"
+              >laptops &amp; computers</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#gadgets">gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#tablets">tablets</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#photo">photo</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#video">video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <article class="card catalog__card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="product"
+        >
+
+        <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <h3 class="card__subtitle">Product code: 195434</h3>
+
+        <div class="card__rate">
+          <div class="stars card__stars stars--4">
+            <div class="stars__star"></div>
```

Comment:
Empty `div`-s is not a good idea for SEO optimization. In this part of your HTML you use a background image for your stars and your image is svg. Here you can use only svg tag for rating system. It's will be better because in the future, when you want to add some animation, animate svg will be easier than simple background image. Furthermore, just because svg is simple html tags you can add classes and modifiers and use your tags like simple html.